pkg/plugins/manager/loader/finder/local.go

```diff
@@ -24,15 +25,21 @@ var (
 )
 
 type Local struct {
-	log log.Logger
+	log     log.Logger
+	devMode bool
```

Comment:
I think it would be better if we just had the cfg as a field here instead. Passing just a boolean to the constructor gives me immediate [Uncle Bob nightmares](https://www.informit.com/articles/article.aspx?p=1392524) 😄